doc/source/management/data_at_rest_encryption.rst

```diff
@@ -477,16 +456,73 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. rubric:: Redo log encryption
+
+Key rotation for the redo log has been redesigned to rotate the key with `SELECT rotate_system_key("percona_redo").
```

Comment:
Since this is in the upgrading section: this means that if somebody used keyring_key encrypted redo logs before 8.0.16, we won't be able to read it. He'll have to turn off encryption, do a normal shutdown, delete the redo logs, and then upgrade. But as this is still a beta (?) feature, we decided that this is ok.

Reply:
I do not want to advertise rotate_system_key functionality - as this is not GAed yet. Only master_key encryption is GAed. Before 8.0.16 I was implementing change to system keys - so the names of those keys included the server's GUID. After this is done we will test redo log with keyring_key encryption.

Reply:
I think keyring_key redo encryption was already mentioned in previous docs, even as it was beta. Encryption threads weren't? (but if we do not want to mention rotate_system_key, we shouldn't mention most redo log changes)

Reply:
Removed - saved for 8.0.17